### ISSUE TITLE
Chart legend does not appear

### ISSUE THREAD

**Issue by pimentelra** [Chris927/react-roughviz#1]
Legend does not appear in charts, even when property legend is set to true.

Tried with the examples on README without success.

```html
<h3>Bar</h3>
<Bar
  data='https://raw.githubusercontent.com/jwilber/random_data/master/flavors.csv'
  labels='flavor'
  values='price'
  legend={true}
/>
<h3>Pie</h3>
<Pie
  data={{
    labels: ['North', 'South', 'East', 'West'],
    values: [10, 5, 8, 3]
  }}
  title='Regions'
  colors={['red', 'orange', 'blue', 'skyblue']}
  roughness={8}
  legend={true}
/>
```

**Comment by Chris927** [comment]
Thanks for reporting this, @pimentelra!

This is fixed in version 0.1.4: I was importing the wrong minified file from the roughviz package, see b189ed7.

**Comment by pimentelra** [comment]
Confirmed it's working now.
Yesterday I went through the code trying to figure out if I could help but, unfortunately, my javascript/ React knowledge is not strong enough.
Thank you so much for correcting the issue.
Congrats on this great project.
